### PR TITLE
[SOTO Update] Shifted the burrito button to the right

### PR DESCRIPTION
**Pull request by klingbolt**
Adjusted the button to account for the new icon

### CODE REVIEW

**Comment by AsherGlick** [comment]
Cant seem to click the burrito icon anymore

File: Spatial.tscn [hunk 1]
```diff
@@ -71,28 +71,27 @@ scale = Vector2( 2, 2 )
 [node name="Line2D" parent="Control/MiniMap" instance=ExtResource( 4 )]
 
 [node name="GlobalMenuButton" type="Control" parent="Control"]
-margin_left = 287.348
-margin_right = 314.348
-margin_bottom = 32.0
+margin_left = 321.348
+margin_top = 4.0
```
Comment (AsherGlick) [inline Spatial.tscn:75]: You took this off of the texture, now it does not line up to the top of the screen.
Screenshot from 2023-08-24 21-12-58

**Review by AsherGlick — CHANGES_REQUESTED** [review]
You took this off of the texture, now it does not line up to the top of the screen.

File: Spatial.gd [hunk 2]
```diff
@@ -119,12 +119,13 @@ func exit_burrito():
 
 
 func set_minimal_mouse_block():
-	var top_corner := Vector2(287, 0)
-	var bottom_corner := Vector2(314, 32)
+	var menu_button = $Control/GlobalMenuButton
+	var top_corner = menu_button.get_position()
+	var bottom_corner = menu_button.get_position() + menu_button.get_size()
```
Comment (klingbolt) [inline Spatial.gd:124]: By changing the values here to the properties the numbers come from, any more changes to the icons will be updated in the passthrough

File: Spatial.tscn [hunk 3]
```diff
@@ -71,38 +71,34 @@ scale = Vector2( 2, 2 )
...
-margin_bottom = 32.0
+margin_right = 29.0
+margin_bottom = 30.0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="EditorQuckPanel" type="PanelContainer" parent="Control/GlobalMenuButton"]
 visible = false
 margin_left = 32.0
```
Comment (klingbolt) [inline Spatial.tscn:101]: I found that the margin besides the left margin does not make a difference if the margin is less than the size. Thus, removing the unnecessary ones

**Comment by AsherGlick** [comment]
Screenshot from 2023-08-28 18-55-11
I think the icon is off still horizontally. My measurements say the icons are 32px apart

**Comment by AsherGlick** [comment]
Screenshot from 2023-08-28 18-59-33
Looks like the height is still 1px off

File: Spatial.tscn [hunk 4]
```diff
@@ -71,38 +71,34 @@ scale = Vector2( 2, 2 )
...
 [node name="main_menu_toggle" type="Button" parent="Control/GlobalMenuButton"]
 modulate = Color( 1, 1, 1, 0 )
+margin_left = -2.0
 margin_right = 27.0
 margin_bottom = 32.0
 __meta__ = {
 "_edit_use_anchors_": false
 }
 
 [node name="EditorQuckPanel" type="PanelContainer" parent="Control/GlobalMenuButton"]
```
Comment (klingbolt) [inline Spatial.tscn:98]:
```suggestion
[node name="EditorQuckPanel" type="PanelContainer" parent="Control/GlobalMenuButton"]
visible = false
```

**Pull request merged**